src/openklant/components/klantinteracties/api/serializers/digitaal_adres.py

```diff
@@ -85,6 +88,7 @@ class Meta:
                 "help_text": _("De unieke URL van dit digitaal adres binnen deze API."),
             },
         }
+        validators = [OptionalEmailValidator()]
```

Comment:
It would be nice if this validation is also done in the admin (not only through the API).

Reply:
I've added this in https://github.com/maykinmedia/open-klant/pull/271/commits/90076edd6199e6c6c79d42260233aa926581d9c5